libs/dh/admin/feature-permissions/src/lib/edit/dh-edit-permission-modal.component.ts

```diff
@@ -52,23 +50,20 @@ import { WattTextAreaFieldComponent } from '@energinet-datahub/watt/textarea-fie
       }
     `,
   ],
-  providers: [DhAdminEditPermissionStore],
   imports: [
+    RxPush,
```

Comment:
nitpick: `RxPush` doesn't seem to be used anymore.

Reply:
Good catch